en/PlanView/pattern_corridor_scan.md

```diff
@@ -1,8 +1,8 @@
 # Corridor Scan (Plan Pattern)
 
-A corridor scan allows you to create a flight pattern that follows a poly-line. 
-This can be used to, for example, survey a road.
-It is supported on all autopilots and vehicle types.
+A corridor scan allows you to create a flight pattern that follows a poly-line. This can be used to, for example, survey a road. It is supported on all autopilots and vehicle types.
+
+> **Important** When planning a Corridor Scan using camera specifications the ground elevations under your survey area are assumed to be at the same altitude as your launch/home location. If the ground elevation under your survey area is either higher or lower than your launch/home location the effective overlap in your images will be less or more (respectively) than calculated. If ground elevation under your survey area is significantly higher than your launch/home location you could inadvertently plan a mission which causes the vehicle to fly into trees, obstacles, or the ground. Use Terrain Follow to create a survey that more closely maintains the desired altitude above terrain that has significant elevation differences from your launch/home altitude.
```

Comment:
See my comment https://github.com/mavlink/qgc-user-guide/pull/263/files#r423459117. You should link down to how they enable terrain following. Ideally you'd copy my pattern in section below, but thats up to you